.gitignore

```diff
@@ -3,3 +3,4 @@
 qmail.plugin
 scanner.plugin
 svstat.plugin
+tags
```

Comment:
My opinion on this is that `.gitignore` should contain only patterns and files generated directly by build system. `tags` is file from external utility specific to your work-flow, therefore it should be in your global ignore list, search `core.excludesFile` in `git-config(1)` if you want to know more.

There is no need to change anything in this commit, I just want to share my point of view.